record-builder-core/src/main/java/io/soabase/recordbuilder/core/RecordBuilder.java

```diff
@@ -94,6 +94,11 @@
          */
         String fromMethodName() default "from";
 
+        /**
+         * The name to use for the from-to-wither inner class
+         */
+        String fromInnerClassName() default "WitherImpl";
```

Comment:
Can you explain this better? I'm -1 on this if its only purpose is to make Jacoco happy.

Reply:
It is not about making Jacoco happy, it is about not including generated code in test coverage.

Reply:
Can you explain in detail please?

Reply:
![image](https://user-images.githubusercontent.com/1169354/155304744-1be20b86-ab19-4d02-8d39-27edfd7e323b.png)
I would like the record-builder generated classes to be excluded from coverage.

Reply:
OK - but can't this be done by configuring Jacoco? Changing the code generation just so code coverage produces better reports seems backwards to me.

Reply:
To me it seems better if library users have to configure their build as little as possible. Excluding generated code from test coverage seems like it should be supported by annotation processor. Lombok does it.